__tests__/integration_testing.test.js

```diff
@@ -102,4 +102,24 @@ describe('getArticles', () => {
             }
         })
     });
+});
+
+describe('getComments', () => {
+    it('should return a 200 status code', () => {
+        return request(app).get("/api/articles/1/comments").expect(200)
+    });
+    it('should return the articles (an array of objects with the correct properties) ', () => {
+        return request(app).get("/api/articles/1/comments").then((res) => {
+            expect(Array.isArray(res.body.comments)).toBe(true)
+            expect(res.body.comments).toHaveLength(11)
+            res.body.comments.forEach((comment, index) => {
+                expect(typeof res.body.comments[index]).toBe('object')
+                expect(comment).hasOwnProperty('author')
+                expect(comment).hasOwnProperty('article_id')
```

Comment:
```suggestion
                expect(comment).hasOwnProperty('article_id')
```
Is having an aritcle_id property enough here or should this be a particular value?